animation_url/src/routes/[slug]/+page.ts

```diff
@@ -0,0 +1,80 @@
+import { connect } from "@tableland/sdk";
+
+const attributesTable = "rig_attributes_80001_3507";
+const lookupsTable = "lookups_80001_3508";
+const pilotSessionsTable = "pilot_sessions_80001_3515";
+const ipfsGatewayUri = "https://nftstorage.link/ipfs/";
+
+/** @type {import('./$types').PageLoad} */
+export async function load({ url }) {
+  const tableland = connect({ chain: "polygon-mumbai" });
+
+  // get rig id, allowing for .html suffix
+  let rigId = url.pathname;
+  const parts = rigId.replace(/^\/|\/$/g, "").split(".");
+  rigId = parts[0];
```

Comment:
Would it make sense to check if we got the rigId here and throw if it's not what we expect?

Reply:
could yeah, `rigId` _will_ be something at this point, but could be a garbage route. i was just letting the sdk return an error from the validator since any true validation would be involved, ie, check it's a number, check it's within range 1-3000. 

<img width="948" alt="Screen Shot 2022-10-17 at 3 44 16 PM" src="https://user-images.githubusercontent.com/361000/196297440-c44f6a62-3756-4015-91ce-c776e84f3223.png">

Reply:
That makes sense